nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/EmailProvenanceReporter.java

```diff
@@ -131,6 +132,13 @@ public class EmailProvenanceReporter extends AbstractProvenanceReporter {
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
+    public static final PropertyDescriptor SPECIFIC_RECIPIENT_ATTRIBUTE_NAME = new PropertyDescriptor.Builder()
+            .name("Specific Recipient Attribute Name")
+            .displayName("Specific Recipient Attribute Name")
```

Comment:
what is the difference between name and displayName? do we need both?

Reply:
the displayName is what appears on the interface when configuring the reporting task properties. In the error messages , the displayName is used that's why I added it. the name is something used internally  while the displayName is what the user sees eventually.